.github/workflows/build.yml

```diff
@@ -5,6 +5,10 @@ on:
   push:
     branches:
       - main
+# the most recent commit in the PR would cancel older commit's in-progress workflows
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number }}
```

Comment:
This still doesn't feel right. Commits in the main branch will all go to the same `build-` group and cancel the previous builds in progress?

Reply:
How about now? https://github.com/NVIDIA/Fuser/actions/workflows/build.yml shows the commit pushes are not cancelled. Only the PR related jobs are cancelled. Again, thanks for raising this!